.openshift-ci/drivers/scripts/kernel-splitter.py

```diff
@@ -0,0 +1,179 @@
+#! /usr/bin/env python3
+
+import os
+import re
+from math import floor
+
+TASKS_DIR = '/tasks'
+
+
+class Task:
+    def __init__(self, kernel, module, driver_type):
+        self.kernel = kernel
+        self.module = module
+        self.driver_type = driver_type
+
+    def parse(task_line: str):
+        task = task_line.strip().split()
+
+        if len(task) != 3:
+            # invalid line
```

Comment:
Maybe print something here.

Reply:
I've print a message if `Task.parse` fails, that way I keep the responsibility of reporting separated from parsing the line, does that work?

Reply:
Yes